packages/core/test/client.test.ts

```diff
@@ -627,6 +629,43 @@ describe('@bugsnag/core/client', () => {
     })
   })
 
+  describe('_isBreadcrumbTypeEnabled()', () => {
+    it.each(breadcrumbTypes)('returns true for "%s" when enabledBreadcrumbTypes is not configured', (type) => {
+      const client = new Client({ apiKey: 'API_KEY_YEAH', enabledBreadcrumbTypes: null })
```

Comment:
Was this intending `enabledBreadcrumbTypes` to be undefined? This test case and the following one seem to be identical.
```suggestion
      const client = new Client({ apiKey: 'API_KEY_YEAH' })
```

Reply:
It was indeed, good spot!